dom/nodes/Document-adoptNode.html

```diff
@@ -47,4 +47,45 @@
   var doc = document.implementation.createDocument(null, null, null)
   assert_throws_dom("NOT_SUPPORTED_ERR", function() { document.adoptNode(doc) })
 }, "Adopting a Document should throw.")
+
+test(function() {
+  var iframe = document.createElement("iframe")
+  document.body.appendChild(iframe)
+  var parent_doc = iframe.contentDocument
+  var doc = parent_doc.body.appendChild(parent_doc.createElement("iframe")).contentDocument
+  assert_not_equals(parent_doc, doc)
+  var y = parent_doc.documentElement
```

Comment:
y is very confusing name here, though looks like the test in general uses confusing variable names.

Reply:
yeah, this is just copy-pasta nonsense…